chromium/qsc-certs.patch

```diff
@@ -1,3 +1,28 @@
+From 021cbbf2e551386c236fb3e9fa97f95be894d278 Mon Sep 17 00:00:00 2001
+From: "Ubuntu 19.10" <57787676+baentsch@users.noreply.github.com>
+Date: Mon, 20 Jul 2020 10:23:43 +0200
+Subject: [PATCH] enable QSC certs
+
+---
+ net/cert/internal/simple_path_builder_delegate.cc | 4 ++--
+ third_party/boringssl/BUILD.gn                    | 5 +++--
+ 2 files changed, 5 insertions(+), 4 deletions(-)
+
+diff --git a/net/cert/internal/simple_path_builder_delegate.cc b/net/cert/internal/simple_path_builder_delegate.cc
+index e4259ee6eb39..b7a79cfef214 100644
+--- a/net/cert/internal/simple_path_builder_delegate.cc
++++ b/net/cert/internal/simple_path_builder_delegate.cc
+@@ -110,8 +110,8 @@ bool SimplePathBuilderDelegate::IsPublicKeyAcceptable(EVP_PKEY* public_key,
+     return true;
+   }
+ 
+-  // Unexpected key type.
+-  return false;
++  // Accept all keys for QSC;
++  return true;
```

Comment:
Accepting all keys seems very permissive. Is there a risk here that too much is being accepted?  Can we add a return true for the OQS algorithms but still have a return false for the remainder?

Reply:
I also thought about that -- but then had to accept I don't understand BoringSSL already good enough to facilitate such check. @xvzcf: Do you have a suggestion?